src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,132 @@
+    const currentValue = event.target.value.trimStart() || null;
+
+    const preparedSearchWith = getSearchWith(
+      searchParams,
+      {
+        query: currentValue,
+      },
+    );
+
+    setSearchParams(preparedSearchWith);
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <SearchLink
+          className={classNames({
+            'is-active': currentSex === null,
+          })}
+          params={{
+            sex: null,
+          }}
+        >
+          All
+        </SearchLink>
+        <SearchLink
+          className={classNames({
+            'is-active': currentSex === 'm',
+          })}
+          params={{
+            sex: 'm',
+          }}
+        >
+          Male
+
+        </SearchLink>
+        <SearchLink
+          className={classNames({
+            'is-active': currentSex === 'f',
+          })}
+          params={{
+            sex: 'f',
+          }}
+        >
+          Female
+
+        </SearchLink>
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
+            data-cy="NameFilter"
+            type="search"
+            className="input"
+            placeholder="Search"
+            value={currentQuery}
+            onChange={handleSearch}
+          />
+
+          <span className="icon is-left">
+            <i className="fas fa-search" aria-hidden="true" />
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {allCenturies.map(century => (
+              <SearchLink
+                data-cy="century"
+                key={century}
+                className={classNames(
+                  'button mr-1',
+                  { 'is-info': centuries.includes(century) },
+                )}
+                params={{
+                  centuries: centuries.includes(century)
+                    ? centuries.filter(c => c !== century)
+                    : [...centuries, century],
+                }}
+              >
+                {century}
+              </SearchLink>
+            ))}
+          </div>
+
+          <div className="level-right ml-4">
+            <SearchLink
+              data-cy="centuryALL"
+              className={classNames(
+                'button is-success',
+                { 'is-outlined': centuries.length },
+              )}
+              params={{
+                centuries: [],
+              }}
+            >
+              All
+            </SearchLink>
+          </div>
+        </div>
+      </div>
+
+      <div className="panel-block">
+        <Link
+          to="/people"
+          className="button is-link is-outlined is-fullwidth"
+        >
+          Reset all filters
+        </Link>
```

Comment:
```suggestion
        <SearchLink
          className="button is-link is-outlined is-fullwidth"
          params={{
            sex: null,
            query: null,
            centuries: null,
          }}
        >
          Reset all filters
        </SearchLink>
```
You need to reset all filters here